src/devices_models/device_constructors/branch_constructor.jl

```diff
@@ -295,6 +311,66 @@ function construct_device!(
     return
 end
 
+function construct_device!(
+    container::OptimizationContainer,
+    sys::PSY.System,
+    ::ModelConstructStage,
+    model::DeviceModel{V, StaticBranch},
+    network_model::NetworkModel{T},
+) where {V <: PSY.ACTransmission, T <: AbstractSecurityConstrainedPTDFModel}
+    devices = get_available_components(model, sys)
+    add_constraints!(container, NetworkFlowConstraint, devices, model, network_model)
+    add_constraints!(container, RateLimitConstraint, devices, model, network_model)
+
+    # TODO: Security constrained. Remove this line. Method not defined
```

Comment:
?

Reply:
@SebastianManriqueM This is a function that doesn't exist